django/db/migrations/operations/special.py

```diff
@@ -100,9 +100,9 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
     def describe(self):
         return "Raw SQL operation"
 
-    def _run_sql(self, schema_editor, sql):
-        if isinstance(sql, (list, tuple)):
-            for sql in sql:
+    def _run_sql(self, schema_editor, sqls):
```

Comment:
I think I'd favor `noop` directly at this level rather than supporting `noop` within the list. So `sql=('', ('something', 'something')` triggers an error rather than just skipping as no-op.